app/frontend/src/pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -0,0 +1,49 @@
+// External Imports
+import React, { Fragment } from "react";
+import { useNavigate } from 'react-router-dom';
+
+// Internal Imports
+import { HeaderNav } from "components/components";
+import { Button } from "components/components";
+import { logoHorizontal, logoStacked, notFoundPageImg } from "assets/images/images";
+
+function NotFoundPage() {
+  const navigate = useNavigate();
+  return (
+    <Fragment>
+      <HeaderNav
+        logoDesktop={logoHorizontal}
+        logoMobile={logoStacked}
+        menu={[
+          { name: "Hack for LA", link: "/" },
+          { name: "How to Join", link: "/qualifier/1" },
+          { name: "Projects", link: "/demo" },
+        ]}
+      />
+      <main>
+        <div className="not-found-container flex-container justify-between align-center gap-3">
+          <div style={{width: "348px"}}>
+            <h1 className="not-found-title mt-0 mb-3">Page not found</h1>
+            <p className="not-found-paragraph mb-3">
+              We can’t seem to find the page you’re looking for.
+              Try going back to the previous page.
+            </p>
+            <Button
+              length="long"
+              size="lg"
+              addClass="landing-intro-btn"
```

Comment:
Hmm, we can probably remove this prop, since we don't care about responsive design at the current moment. It is also name-spaced to the landing page.

That said, once we're ready to open a discussion on responsive designs, we can probably discuss the best way to create and maintain global responsive components independent of page namespaces.

Thoughts?

Reply:
Yep removing `addClass` prop makes sense here

Not too sure about the best method for re-organizing to global responsive components but am excited to learn!